src/CruxComponent.tsx

```diff
@@ -190,35 +194,47 @@ export class CruxComponentCreator {
                             <Table className="table table-striped cftable" striped bordered condensed hover>
                                 <thead>
                                 <tr>
-                                    {_.map(constants.fields.filter((field: any) => field.display), (field: any, index: any) =>
+                                    {constants.fields.filter((field: any) => field.display).map((field: any, index: any) =>
                                         <th key={index}>{field.title}</th>)}
                                     {constants.editModal && <th></th>}
                                 </tr>
                                 </thead>
                                 <tbody>
-                                {_.map(filteredRows,
-                                    (model: any, index: number) => {
-                                        const filtered = constants.fields.filter((field: any) => field.display === true)
-                                        return <tr key={index}>
-                                            {_.map(filtered, (field: any, i: number) => {
-                                                    return <td key={i}
-                                                               style={(field.cellCss) ? field.cellCss : { margin: "0px" }}>
-                                                        <div style={{ marginTop: 8 }}>
-                                                            <ListNestedComponent
-                                                                field={field} model={model}
-                                                                additionalModels={this.props.additionalModels}
-                                                                modelChanged={this.inlineEdit} />
-                                                        </div>
-                                                    </td>
-                                                }
-                                            )}
-                                            {constants.editModal &&
-                                            <td key={2}><span style={{ marginLeft: "20px", marginTop: 8, color: "grey" }}
-                                                              className="glyphicon glyphicon-pencil fas fa-pencil-alt"
-                                                              aria-hidden="true" onClick={this.showEditModal(model)} />
-                                            </td>}
-                                        </tr>
+                                <tr key='searchRow'>
```

Comment:
adding `search-row` at the top for all fields that have enabled field level search